src/utils/actions.js

```diff
@@ -296,65 +298,119 @@ const actions = [
     translation: 'Object.ActionButton.ExecuteAutomation',
     value: 'executeAutomation',
     getActionCall:
-      ({ app, automation, automationPostData }) =>
-      async () => {
-        if (automation !== undefined) {
-          try {
-            automation = encodeURIComponent(automation);
-            const itemInfo = await fetch(`../api/v1/items/${automation}`).then((response) => response.json());
-            const autoInfo = await fetch(`../api/v1/automations/${itemInfo.resourceId}`).then((response) =>
-              response.json()
-            );
-            let executePath = `../api/v1/automations/${autoInfo.id}/actions/execute?X-Execution-Token=${autoInfo.executionToken}`;
+      ({ app, automation, automationId, automationTriggered, automationExecutionToken, automationPostData, buttonId, multiUserAutomation }) =>
```

Comment:
please change the comment above to reflect the new arguments

Reply:
done in the latest commit